openedx/features/enterprise_support/tests/test_api.py

```diff
@@ -421,26 +421,50 @@ def fake_reverse(*args, **kwargs):
             [
                 'If you have concerns about sharing your data, please contact your administrator at GriffCo.',
                 'Enrollment in My Cool Course was not complete.'
-            ]
+            ],
+            "", ""
+        ),
+        (
+            True,
+            {'name': 'GriffCo', 'uuid': 'real-uuid'},
+            'real-course',
+            [
+                MockEnrollment(
+                    course_id='real-course',
+                    course_overview=mock.MagicMock(
+                        display_name='My Cool Course'
+                    )
+                )
+            ],
+            [
+                'If you have concerns about sharing your data, please contact your administrator at GriffCo.',
+                'Enrollment in My Cool Course was not complete.'
+            ],
+            "Title from DataSharingConsentPage modal", "Message from DataSharingConsentPage modal"
```

Comment:
nit: May be we should change these text to something
`Title from DataSharingConsentTextOverrides model in consent app`
and 
`Message from DataSharingConsentTextOverrides model in consent app`